osbrain/agent.py

```diff
@@ -204,6 +214,19 @@ class Agent():
         When set to `True`, the agent will continue executing the main loop.
     running : bool
         Set to `True` if the agent is running (executing the main loop).
+    _async_req_uuid : dict
+        Stores a unique identifier for every REQ or PULL connection.
+    _async_req_handler : dict
+        Stores the handler for every active REQ or PULL connection.
```

Comment:
I think these `_async_req_*` are incorrect, but would need to have a look at the code to know what they are exactly doing. Maybe it is not the best name, as they were probably initially created for ASYNC-REQ channels but are now used for SYNC-PUB channels as well...

Reply:
This is all I could guess from reading the code. If this is not what they are really used for, I will need a more detailed description from you 😄

Reply:
Maybe:

- Stores the UUIDs of the asynchronous request sockets (used in communication channels).
- Stores the handler for every asynchronous request sockets (used in communication channels).

Reply:
Yeah, I'll definitely need to read a bit more on how channels work here.